### ISSUE TITLE
quick setup instructions

### ISSUE THREAD

**Issue by hxrts** [mhfowler/open-source-feeds#1]
I get most of what's going on but not entirely sure how you're organizing the directory structure.

**Comment by mhfowler** [comment]
so far I was just learning docker and experimenting a bit so there is no organization currently -- I'll clean it up once I get a bit further and then will definitely add a readme (also +1 to using github issues)

**Comment by hxrts** [comment]
yeah, I figured you were just messing with a few things at this point. will keep an eye out for the updated readme.
